.jsonschema.json

```diff
@@ -145,11 +169,62 @@
               "additionalProperties": false
             }
           ]
-
         }
       },
       "additionalProperties": false
     },
+    "duplicate": {
+      "$id": "#duplicate",
+      "description": "A \"dup\" brand",
+      "type": "object",
+      "required": ["title"],
+      "properties": {
+        "title": {
+          "$ref": "#/definitions/title"
+        },
+        "slug": {
+          "$ref": "#/definitions/slug"
+        },
+        "hex": {
+          "$ref": "#/definitions/hex"
+        },
+        "guidelines": {
```

Comment:
Do we want to allow for a `source` here as well, for the `hex`? Think we discussed that briefly in the issue.

Reply:
[You brought up that we didn't need it](https://github.com/simple-icons/simple-icons/issues/4797#issuecomment-772705615)  and [I agreed](https://github.com/simple-icons/simple-icons/issues/4797#issuecomment-772829551).

Thinking about it again, though I think it is unlikely the duplicates hex is a different source, it's certainly possible so probably better to put it in - at the risk of having duplicate sources (which we _could_ lint for, but that seems very low priority to me :thinking:).